templates/cli/lib/commands/push.js.twig

```diff
@@ -1101,46 +1101,52 @@ const pushMessagingTopic = async ({ all, yes } = {}) => {
 }
 
 const push = new Command("push")
+    .alias("deploy")
```

Comment:
I dont think we should keep `deploy` . It will continue to remind people that deploy exists and will prevent us from completing a proper migration to use `push` and `pull`

Reply:
Removed